.github/workflows/msys2_ffmpeg.yml

```diff
@@ -0,0 +1,129 @@
+name: MSYS2 FFmpeg Plugins
+on:
+  push:
+    paths:
+      - .github/workflows/msys2_ffmpeg.yml
```

Comment:
This workflow only run when there's a change on `.github/workflows/msys2_ffmpeg.yml`, is this a designed behavior?

Reply:
It can be changed to cater for 

on: [push, pull_request, workflow_call] 

Hence for every changes it will trigger the following composite action to generate FFmpeg v4.4.2 open source with IMTL